specification/_json_spec/inference.unified_inference.json

```diff
@@ -0,0 +1,45 @@
+{
+  "inference.unified_inference": {
+    "documentation": {
+      "url": "https://www.elastic.co/guide/en/elasticsearch/reference/master/unified-inference-api.html",
+      "description": "Perform inference using the Unified Schema"
+    },
+    "stability": "stable",
+    "visibility": "public",
+    "headers": {
+      "accept": ["text/event-stream"],
```

Comment:
This is what we have for the streaming api here: https://github.com/elastic/elasticsearch-specification/blob/main/specification/_json_spec/inference.stream_inference.json#L10

Not sure if this is correct if it's indicating that the post request accepts an event stream or that the response returns one 🤔

Reply:
Both HTTP headers are used in client requests:

 * Accept tells Elasticsearch that returning event streams is OK
 * Content-Type is about the request's body, which is indeed JSON here.

In other words, this is correct, thank you!